### ISSUE TITLE
allow multiple source location in gradle

### ISSUE THREAD

**Issue by tbee** [asciidoctor/asciidoclet#67]
I use asciidoctlet in JFXtras and am very pleased with it. Recently Maven central required all jars to be accompanied with a source jar and javadoc jar. Beside the component jars, JFXtras also releases an all-in-one jar, which now requires a source and javadoc jar. I'm not very fluent in Gradle, but apparently asciidoctor cannot use multiple source locations so to generate one big javadoc. Is this easily added?

**Comment by johncarl81** [comment]
I believe we're deploying the source and javadoc jars.  For example: https://bintray.com/asciidoctor/maven/asciidoclet#files/org%2Fasciidoctor%2Fasciidoclet%2F1.5.6

**Comment by tbee** [comment]
The operative word here being "multiple".

**Comment by msgilligan** [comment]
Look at the ConsensusJ project for an example...

**Comment by johncarl81** [comment]
Correct me if I'm wrong @msgilligan... here's the configuration @tbee to collect all your project source javadocs together: https://github.com/ConsensusJ/consensusj/blob/master/gradle/javadoc.gradle#L49-L50

This is less of a doclet issue than a Javadoc thing.

**Comment by msgilligan** [comment]
Yes, @johncarl81 it's a Gradle and Javadoc configuration issue, but ConsensusJ shows one way of doing it.

BTW, it's really cool to hear that https://github.com/JFXtras is using Asciidoclet!

**Comment by johncarl81** [comment]
@tbee, does this give you enough to go on?
